bip-0099.mediawiki

```diff
@@ -25,19 +25,17 @@ deployment of changes.
 
 ==Defintions==
 
-- Software fork: in free software, an existing project can be copied
-  and extended/maintained without permission from the original
-  project's maintainers.
-- Consensus fork: a divergence in the implementation of the
-  verification rules can impede the expected eventual convergence of
-  the network in a single chain that has the most proof of work and
-  also satisfies the rules. This can be intentional or be caused by a
-  bug in consensus validation reimplementations.
-- Softfork: an intentional consensus fork where everything that
-  was previously invalid remains invalid, but some things that were
-  valid also become invalid. Majority of miners can impose
-  this kind of changes and they also have some deployment advantages.
-- Hardfork: an intentional consensus fork that is not a softfork.
+;Software fork
+: A copy of an existing project where management is separate from the existing project.
+
+;Consensus fork
+: A divergence in the implementation of the verification rules can impede the expected eventual convergence of the network in a single chain that has the most proof of work and also satisfies the rules. This can be intentional or be caused by a bug in consensus validation reimplementations.
```

Comment:
is there a reason to use long single lines? It makes it harder to correct things in parallel on the same text (not a huge advantage for multiple lines, but, still, an advantage).